docs/source/io.rst

```diff
@@ -61,4 +61,11 @@ TIFF Reader/Writer
    :members:
    :special-members:
 
-:ref:`Return Home <mastertoc>`
+RAW File Writer
+===============
+
+.. autoclass:: cil.io.RAWFileWriter
```

Comment:
Why does the `write` method not appear in the documentation?
I think it should

Reply:
It doesn't appear in the TIFF writer docs either 😞

Reply:
maybe need to add a docstring to the `write` method to fix this problem

Reply:
added that